src/styles/blocks/header.scss

```diff
@@ -0,0 +1,7 @@
+* {
+  box-sizing: border-box;
```

Comment:
The 'box-sizing' property should be in a separate file (e.g., 'base.scss') and not in 'header.scss', as it is a global style. Rename this file into e.g `base.scss`